pkg/api/types.go

```diff
@@ -601,6 +601,12 @@ type EmptyDirVolumeSource struct {
 	// The default is "" which means to use the node's default medium.
 	// +optional
 	Medium StorageMedium
+	// Total amount of local storage required for this directory.
+	// The size limit is also applicable for Memory type.
+	// The default is nil which means that the limit is undefined.
+	// More info: http://kubernetes.io/docs/user-guide/volumes#emptydir
+	// +optional
+	SizeLimit resource.Quantity
```

Comment:
Yes, the total memory request is still 500Mi. When specifying the size limit for emptydir, it does not change pod qos